### PR TITLE
[DO NOT MERGE] add test tag rocm_multi_gpu, run those separately for CSB builds

### PR DESCRIPTION
**Pull request by jeffdaily**
Once approved, this PR will be submitted against TF master.

### CODE REVIEW

**Comment by whchung** [comment]
@sunway513  could you point CSB CI build to give this PR a spin?

**Comment by sunway513** [comment]
@whchung , let's track this CI run: 
http://ml-ci.amd.com:21096/job/tensorflow-rocm-nightly-debug/14/

File: tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh [hunk 1]
```diff
@@ -51,6 +52,20 @@ bazel test \
       -//tensorflow/contrib/... \
       -//tensorflow/lite/... \
       -//tensorflow/python/compiler/tensorrt/... \
-
-
+&& \
+bazel test \
+      --config=rocm \
+      -k \
+      --test_tag_filters=rocm_multi_gpu, \
```
Comment (deven-amd) [inline tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: need to add `-no_rocm` here

Comment (jeffdaily) [reply tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: Is that strictly true?  `rocm_multi_gpu` is already specific to ROCm.  Doesn't the `--test_tag_filters` whitelist only the `nccl_manager_test` with this tag?  Perhaps I misunderstood what tags can do.

Comment (deven-amd) [reply tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: think about a future scenario where you have multiple tests with `rocm_mutli_gpu` tag and one of them needs to be disabled for some reason....

Comment (deven-amd) [reply tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: another way to think about it is that adding `no_rocm` tag should always disable that test from running on the CI....so all `bazel test` commands must filter those tests out

**Comment by jeffdaily** [comment]
The failing XLA tests don't seem related to my change.  Why are these failing now?

```
//tensorflow/compiler/tests:categorical_op_test_gpu
//tensorflow/compiler/tests:image_ops_test_gpu
//tensorflow/compiler/tests:listdiff_op_test_gpu
//tensorflow/compiler/tests:pooling_ops_3d_test_gpu
//tensorflow/compiler/tests:reduce_ops_test_gpu
//tensorflow/compiler/tests:self_adjoint_eig_op_test_gpu
//tensorflow/compiler/tests:spacetobatch_op_test_gpu
//tensorflow/compiler/tests:stateless_random_ops_test_gpu
//tensorflow/compiler/xla/service/gpu/tests:gpu_kernel_tiling_test
//tensorflow/compiler/xla/tests:array_elementwise_ops_test_gpu
//tensorflow/compiler/xla/tests:batch_normalization_test_gpu
//tensorflow/compiler/xla/tests:broadcast_simple_test_gpu
//tensorflow/compiler/xla/tests:concat_test_gpu
//tensorflow/compiler/xla/tests:cpu_gpu_fusion_test_gpu
//tensorflow/compiler/xla/tests:dot_operation_single_threaded_runtime_test_gpu
//tensorflow/compiler/xla/tests:dot_operation_test_autotune_disabled_gpu
//tensorflow/compiler/xla/tests:dot_operation_test_gpu
//tensorflow/compiler/xla/tests:dynamic_ops_test_gpu
//tensorflow/compiler/xla/tests:multioutput_fusion_test_gpu
//tensorflow/compiler/xla/tests:pad_test_gpu
//tensorflow/compiler/xla/tests:reduce_test_gpu
//tensorflow/compiler/xla/tests:reduce_window_test_gpu
//tensorflow/compiler/xla/tests:select_and_scatter_test_gpu
//tensorflow/compiler/xla/tests:select_test_gpu
//tensorflow/compiler/xla/tests:tuple_test_gpu
//tensorflow/compiler/xla/tests:vector_ops_simple_test_gpu
```

Plus, our internal debug CI build # 14 failed, as well, and a test I haven't seen fail before.
`//tensorflow/core/grappler/optimizers:remapper_test_gpu`

**Comment by deven-amd** [comment]
finally somebody other than me has hit the 25 failures in the `rocm-xla` 
( see https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/pull/632#issuecomment-530069172 )

They are intermittent failures, that are not reproducible locally. They seem to be happennig frequently enough now to warrant further investigation.

for future reference the failing run is at
http://ml-ci.amd.com:21096/job/tensorflow-upstream-unit-tests-rocm-xla/679/

I will request jenkins to re-run `rocm-xla` 

> The failing XLA tests don't seem related to my change. Why are these failing now?
> 
> ```
> //tensorflow/compiler/tests:categorical_op_test_gpu
> //tensorflow/compiler/tests:image_ops_test_gpu
> //tensorflow/compiler/tests:listdiff_op_test_gpu
> //tensorflow/compiler/tests:pooling_ops_3d_test_gpu
> //tensorflow/compiler/tests:reduce_ops_test_gpu
> //tensorflow/compiler/tests:self_adjoint_eig_op_test_gpu
> //tensorflow/compiler/tests:spacetobatch_op_test_gpu
> //tensorflow/compiler/tests:stateless_random_ops_test_gpu
> //tensorflow/compiler/xla/service/gpu/tests:gpu_kernel_tiling_test
> //tensorflow/compiler/xla/tests:array_elementwise_ops_test_gpu
> //tensorflow/compiler/xla/tests:batch_normalization_test_gpu
> //tensorflow/compiler/xla/tests:broadcast_simple_test_gpu
> //tensorflow/compiler/xla/tests:concat_test_gpu
> //tensorflow/compiler/xla/tests:cpu_gpu_fusion_test_gpu
> //tensorflow/compiler/xla/tests:dot_operation_single_threaded_runtime_test_gpu
> //tensorflow/compiler/xla/tests:dot_operation_test_autotune_disabled_gpu
> //tensorflow/compiler/xla/tests:dot_operation_test_gpu
> //tensorflow/compiler/xla/tests:dynamic_ops_test_gpu
> //tensorflow/compiler/xla/tests:multioutput_fusion_test_gpu
> //tensorflow/compiler/xla/tests:pad_test_gpu
> //tensorflow/compiler/xla/tests:reduce_test_gpu
> //tensorflow/compiler/xla/tests:reduce_window_test_gpu
> //tensorflow/compiler/xla/tests:select_and_scatter_test_gpu
> //tensorflow/compiler/xla/tests:select_test_gpu
> //tensorflow/compiler/xla/tests:tuple_test_gpu
> //tensorflow/compiler/xla/tests:vector_ops_simple_test_gpu
> ```
> 
> Plus, our internal debug CI build # 14 failed, as well, and a test I haven't seen fail before.
> `//tensorflow/core/grappler/optimizers:remapper_test_gpu`

**Comment by deven-amd** [comment]
jenkins : retest rocm-xla please

File: tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh (hunk 1 above)
Comment (jeffdaily) [reply tensorflow/tools/ci_build/linux/rocm/run_csb_tests.sh]: Force pushed after rebase.  Added `-no_rocm` to test tag filter for the `rocm_multi_gpu` case.

**Comment by ROCm-Apps-Test** [comment]
Can one of the admins verify this patch?

**Comment by ekuznetsov139** [comment]
is this still relevant?

**Comment by deven-amd** [comment]
closing PR as it is no longer relevant

**Pull request closed without merging**